static/js/controllers/inbox.js

```diff
@@ -657,6 +658,7 @@ var feedback = require('../modules/feedback'),
         RecurringProcessManager.stopUpdateRelativeDate();
       });
 
+      ClientSentinel.start();
```

Comment:
IMO we should only start this for users with the correct permission. You should add a permission for wealth quintiles and gate the start of this behind that permission. E.g.

```js
Auth('can_get_wealth_quintiles') // it's a terrible name, I'll leave that to you
  .then(function() {
    ClientSentinel.start();
  })
  .catch(function() {});
```

Reply:
You could also consider altering the `Auth` service so there is a function that doesn't throw an exception every time we check something.

Either across the board (though maybe we rely on it in some places), or add an `Auth.check()` function or something.

Reply:
So the main thing that put me off having this as a permission was that it doesn't really seem to be what permissions are for. I guess if various IN stuff is going to live in permissions then that's just the way things are going, but so far it seems permissions are for allowing particularly users access to various functions/views in the app, whereas this is more like a feature flag, which I believe some work is happening around currently?